1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/features/module/modules/misc/RichPresence.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * LiquidBounce Hacked Client
+ * A free open source mixin-based injection hacked client for Minecraft using Minecraft Forge.
+ * https://github.com/CCBlueX/LiquidBounce/
+ */
+package net.ccbluex.liquidbounce.features.module.modules.misc
+
+import net.ccbluex.liquidbounce.LiquidBounce
+import net.ccbluex.liquidbounce.features.module.Module
+import net.ccbluex.liquidbounce.features.module.ModuleCategory
+import net.ccbluex.liquidbounce.features.module.ModuleInfo
+import net.ccbluex.liquidbounce.utils.ClientUtils
+import kotlin.concurrent.thread
+
+@ModuleInfo(name = "RichPresence", description = "Show discord presence", category = ModuleCategory.MISC)
```

Comment:
I don't like the idea of rich presence being a module. Their should be a menu for it somewhere else.

Reply:
The question is: Where?

Reply:
Maybe in the Main menu under Mods?